### PR TITLE
chore(npm): update package name

### PR DESCRIPTION
**Pull request by nrod80**
Is there anything else necessary for configuration here?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "prop-types",
+  "name": "@catalytic/prop-types",
   "version": "15.6.0",
   "description": "Runtime type checking for React props and similar objects.",
   "main": "index.js",
```
Comment (gtowne) [inline package.json:5]: Might want to update `license` to our generic copyright.

Comment (gtowne) [reply package.json:5]: And `repository` and `homepage` to this repository.

**Review by gtowne — APPROVED** [review]
Let me know if you run into permissions issues with this: https://www.npmjs.com/docs/orgs/publishing-an-org-scoped-package.html

File: package.json (hunk 1 above)
Comment (nrod80) [reply package.json:5]: 👍  good call, updated both

**Pull request merged**